mission_control/navigator_launch/nodes/network_listener.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+import rospy
+from std_msgs.msg import String
+from sub8_alarm import single_alarm
+from twisted.internet import reactor
+import time
+
+
+class NetworkCheck(object):
+    '''
+    Meant to only run on the boat. When the network is dropped, it triggers a kill.
+    '''
+    def __init__(self, timeout=2.0):
+        self.timeout = rospy.Duration(timeout)
+        self.sub = rospy.Subscriber('/keep_alive', String, self.got_network_msg, queue_size=1)
+
+        self.alarm_broadcaster, self.alarm = single_alarm('kill', severity=3, problem_description="Network loss")
+        rospy.Timer(rospy.Duration(0.1), self.check)
+
+    def check(self, *args):
+        if self.need_kill() and self.last_msg != '':\
```

Comment:
Won't self.last_msg always == 'keep_alive' after the first call to got_network_msg?
What is this backslash for?